pkgs/by-name/ta/tabby-terminal/package.nix

```
@@ -0,0 +1,64 @@
+{ stdenv, fetchurl, pkgs, lib, alsa-lib }:
```

```suggestion
{ stdenv, fetchurl, lib, alsa-lib }:
```
Definitely don't pass `pkgs` in; pass in the dependencies invididually.